decidim-accountability/app/views/decidim/accountability/results/_project.html.erb

```diff
@@ -40,4 +17,19 @@
   <div class="mt-8">
     <%= cell "decidim/accountability/result_metadata", result, template: :show_footer %>
   </div>
+
+  <% content_for :drawer_aside do %>
+    <div class="accountability__project-aside">
+      <% if component_settings.display_progress_enabled? %>
+        <%= cell(
+          "decidim/accountability/status",
+          result,
+          title: t("models.result.fields.progress", scope: "decidim.accountability"),
+          extra_classes: "accountability__status__project"
+        ) %>
+      <% end %>
+
+    <%= cell("decidim/accountability/result_metadata", result, template: :project_aside) %>
```

Comment:
It seems e.g. the "see other versions" link is not working. Is it intentional?

Reply:
#10174